src/Goerli-DssSpellCollateral.sol

```diff
@@ -31,58 +89,277 @@ contract DssSpellCollateralAction {
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    uint256 constant ZERO_PCT_RATE           = 1000000000000000000000000000;
+    uint256 constant ZERO_ZERO_FIVE_PCT_RATE = 1000000000015850933588756013;
+
+    // --- Math ---
+    uint256 public constant THOUSAND = 10**3;
+    uint256 public constant MILLION  = 10**6;
+    uint256 public constant WAD      = 10**18;
+    uint256 public constant RAY      = 10**27;
+    uint256 public constant RAD      = 10**45;
+
+    // -- RWA008 MIP21 components --
+    address constant RWA_URN_PROXY_ACTIONS   = 0x741A0fc23AB643B1963E77B5136554b12a91f170;
+
+    address constant RWA008                  = 0x30434AA15F85598F45406d08A79dCdD9a79335e9;
+    address constant MCD_JOIN_RWA008_A       = 0x4ce65E856f824C2b4a2eeD88E79b839eB366967d;
+    address constant RWA008_A_URN            = 0x6351915f840937Edba75656727f20165185FaB83;
+    address constant RWA008_A_INPUT_CONDUIT  = 0x6298a2498b4E3789bb8220Cf5f5b44f518509e2a;
+    address constant RWA008_A_OUTPUT_CONDUIT = 0x1FE95E519F0CE8eCF1cdC885f4DeA7913e146149;
+    // SocGen's wallet
+    address constant RWA008_A_OPERATOR       = 0x3F761335890721752476d4F210A7ad9BEf66fb45;
+    // DIIS Group wallet
+    address constant RWA008_A_MATE           = 0xb9444802F0831A3EB9f90E24EFe5FfA20138d684;
+
+    uint256 constant RWA008_A_INITIAL_DC     = 30 * MILLION * RAD;
+    uint256 constant RWA008_A_INITIAL_PRICE  = 30_437_069 * WAD;
+    uint48 constant  RWA008_A_TAU            = 0;
+
+    uint256 constant RWA008_REG_CLASS_RWA    = 3;
+
+    /**
+     * @notice MIP13c3-SP4 Declaration of Intent & Commercial Points -
+     *   Off-Chain Asset Backed Lender to onboard Real World Assets
+     *   as Collateral for a DAI loan
+     *
+     * https://ipfs.io/ipfs/QmdmAUTU3sd9VkdfTZNQM6krc9jsKgF2pz7W1qvvfJo1xk
+     */
+    string constant RWA008_DOC               = "IPFS_HASH"; // Reference to a documents which describe deal (should be uploaded to IPFS)
```

Comment:
Missing: RWA008 Doc IPFS Hash

Reply:
We are still trying to figure out the best way to include the required information on IPFS or similar solution. Since this is a Goerli spell, we believe this document is not too important in order to block a merge. I'll keep you posted.

Reply:
Consider using [pinata](https://www.pinata.cloud/) for uploading on IPFS.

Reply:
The real issue is that these deals are made of several documents. We are still looking for the authoritative source containing all of them.

I have found a [forum post](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264?u=miguel_cabeza) that fills that role for HVB and exported its raw content to upload to IPFS.

We are still missing that for SocGen.